src/__tests__/createGetDataElementValue.test.js

```diff
@@ -188,38 +188,6 @@ describe('function returned by createGetDataElementValue', function () {
     expect(value).toBe('cleaned:bar');
   });
 
-  it(
```

Comment:
This test was moved into the looping tests for `null` and `undefined` to make sure it works for both cases.